src/pymor/vectorarrays/list.py

```diff
@@ -23,6 +23,10 @@ class VectorInterface(BasicInterface):
     def make_zeros(cls, subtype=None):
         pass
 
+    @classmethod
+    def from_data(cls, data, subtype):
+        pass
```

Comment:
This should probably also `raise NotImplementedError`. I believe the test failures on travis are due to this line (FenicsVector does not implement `from_data`).